.buildkite/bk.fips-integration.pipeline.yml

```diff
@@ -0,0 +1,178 @@
+        command: |
+          buildkite-agent artifact download build/distributions/** . --step 'packaging-ubuntu-x86-64-fips'
+          .buildkite/scripts/steps/integration_tests_tf.sh {{matrix}} false
+        artifact_paths:
+          - build/**
+          - build/diagnostics/**
+        retry:
+          automatic:
+            limit: 1
+        agents:
+          provider: "gcp"
+          machineType: "n1-standard-8"
+          image: "${IMAGE_UBUNTU_2404_X86_64}"
+        matrix:
+          - default
+
+      - label: "fips:sudo:{{matrix}}"
+        depends_on:
+          - packaging-ubuntu-x86-64-fips
+        command: |
+          buildkite-agent artifact download build/distributions/** . --step packaging-ubuntu-x86-64-fips
+          .buildkite/scripts/steps/integration_tests_tf.sh {{matrix}} true
+        artifact_paths:
+          - build/**
+          - build/diagnostics/**
+        retry:
+          automatic:
+            limit: 1
+        agents:
+          provider: "gcp"
+          machineType: "n1-standard-8"
+          image: "${IMAGE_UBUNTU_2404_X86_64}"
+        matrix:
+          - default
+          #- upgrade
+          #- upgrade-flavor
+          #- standalone-upgrade
+          #- fleet
+          #- fleet-endpoint-security
+          #- fleet-airgapped
+          #- fleet-airgapped-privileged
+          #- fleet-privileged
+          #- fleet-upgrade-to-pr-build
+          #- install-uninstall
+          #- fqdn
+          #- deb
+          #- container
+
+  #- group: "Kubernetes"
+  #  key: integration-tests-kubernetes
+  #  depends_on:
+  #    - integration-ess
+  #    - packaging-containers-x86-64
+  #  steps:
+  #    - label: "{{matrix.version}}:amd64:{{matrix.variants}}"
+  #      env:
+  #        K8S_VERSION: "{{matrix.version}}"
+  #        ASDF_KIND_VERSION: "0.27.0"
+  #        DOCKER_VARIANTS: "{{matrix.variants}}"
+  #        TARGET_ARCH: "amd64"
+  #        AGENT_VERSION: "9.0.0-SNAPSHOT" # Remove agent pinning once 9.0.0 is released
+  #      command: |
+  #        buildkite-agent artifact download build/distributions/*-linux-amd64.docker.tar.gz . --step 'packaging-containers-x86-64'
+  #        .buildkite/scripts/steps/integration_tests_tf.sh kubernetes false
+  #      artifact_paths:
+  #        - build/**
+  #        - build/diagnostics/**
+  #        - build/*.pod_logs_dump/*
+  #      retry:
+  #        automatic:
+  #          limit: 1
+  #      agents:
+  #        provider: "gcp"
+  #        machineType: "n1-standard-4"
+  #        image: "${IMAGE_UBUNTU_2404_X86_64}"
+  #        diskSizeGb: 80
+  #      matrix:
+  #        setup:
+  #          variants:
+  #          - "basic,slim,complete,service,elastic-otel-collector"
+  #          - "wolfi,slim-wolfi,complete-wolfi,elastic-otel-collector-wolfi"
+  #          version:
+  #          - v1.27.16
+  #          - v1.28.9
+  #          - v1.29.8
+  #          - v1.30.8
+  #          - v1.31.0
+  #          - v1.32.0
```

Comment:
Should this commented out block be removed? If not, maybe add a comment on top of the block explaining why it's commented out for now?

Reply:
I haven't looked into our k8s tests, we would need to pass the FIPS image we create as the `DOCKER_VARIANTS` arg